iterator.test.js

```diff
@@ -0,0 +1,68 @@
+import { describe, expect, test } from "vitest";
+
+import { alphabetIterator } from "./iterator";
+
+/**
+ * Test suite for the iterator functionality.
+ */
+describe("Assignment questions test on iterators", () => {
+  /**
+   * Test case for iterating through alphabets using a while loop.
+   */
+  test("Test for iterating through alphabets using while loop", () => {
+    const iterator = alphabetIterator[Symbol.iterator]();
+    let expectedCharCode = 97;

```

Comment:
We can avoid hardcoding by syaing `'a'.codePointAt(0)`

Reply:
Addressed the changes.